src/domain/spaces/entities/space.entity.ts

```diff
@@ -21,7 +22,7 @@ export const SpaceSchema: z.ZodType<
     safes?: DbSpace['safes'];
   }
 > = RowSchema.extend({
-  name: z.string().max(255),
+  name: NameSchema,
```

Comment:
This is one of the meaningful changes in the PR: `Space` names where plain strings.

Reply:
The database entity has a max limit of 255, whereas `NameSchema` has 20. Should we also change the schema, or increase the schema? We need to take into consideration that there may already be names in the database that are >20 characters in length.

Reply:
I've added the restriction to the database schema in [92dd3f0](https://github.com/safe-global/safe-client-gateway/pull/2513/commits/92dd3f0c910ff4220f233a85bce8817ca49c888f)

In addition to that, I've centralized the min/max name values into `name.schema.ts`, and extended the allowed chars to 30. I think it's a better balance between security and UX. Given that non-ascii and non-alphanumeric characters are disallowed, the 30-characters limitation should be OK, wdyt?.

Reply:
I think the restrictions make sense. Thank you for making the changes.